### PR TITLE
Index/Partners pages with logic and resources added

### PR DESCRIPTION
**Pull request by zachepilkin**
- views/controllers/resources modified 
- sample banners added

### CODE REVIEW

File: AzureDay.WebApp/AzureDay.WebApp.Localization/Host/Views/Shared/_Footer.ru.resx [hunk 1]
```diff
@@ -118,10 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Address" xml:space="preserve">
-    <value>Адрес: г. Киев, ул. Госпитальная 12, President Hotel</value>
+    <value>Адрес: г. Киев, ул. улица, Hotel</value>
```
Comment (boyko-ant) [inline AzureDay.WebApp/AzureDay.WebApp.Localization/Host/Views/Shared/_Footer.ru.resx]: address is available here https://azureday.global/conferences/2018/ua.html

File: AzureDay.WebApp/AzureDay.WebApp.Localization/Host/Views/Shared/_Footer.uk-UA.resx [hunk 2]
```diff
@@ -118,10 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Address" xml:space="preserve">
-    <value>Адреса: м. Київ, вул. Госпітальна, 12, President Hotel</value>
+    <value>Адреса: м. Київ, вул. вулиця, дім, Hotel</value>
```
Comment (boyko-ant) [inline AzureDay.WebApp/AzureDay.WebApp.Localization/Host/Views/Shared/_Footer.uk-UA.resx]: address is available here https://azureday.global/conferences/2018/ua.html

File: AzureDay.WebApp/AzureDay.WebApp.Localization/Host/Views/Shared/_Footer.resx [hunk 3]
```diff
@@ -118,10 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Address" xml:space="preserve">
-    <value>Kyiv, Hospitalna str. 12, President Hotel</value>
+    <value>Kyiv, Street str. Building, Hotel</value>
```
Comment (boyko-ant) [inline AzureDay.WebApp/AzureDay.WebApp.Localization/Host/Views/Shared/_Footer.resx]: address is available here https://azureday.global/conferences/2018/ua.html

**Comment by zachepilkin** [comment]
Venue address added

**Pull request merged**